demos/wascc/fileserver/README.md

```diff
@@ -0,0 +1,22 @@
+# Fileserver
+
+An example that will respond with the file metadata present in the volume, based on the URL path
+provided.
+
+If a POST request is made, the contents of the request body are written to the file based on the URL
+path provided.
+
+If a DELETE request is made, the file is removed based on the URL path provided.
+
+It is meant to demonstrate how volumes work with the wascc-provider.
+
+## Running the example
+
+This example has already been pre-built, so you only need to install it into your Kubernetes
+cluster.
+
+Create the pod and configmap with `kubectl`:
+
+```shell
+$ kubectl create -f k8s.yaml
+```
```

Comment:
With all the different `make` targets and other goodies you added in here, I'd really love to see some building instructions with the different toolsets I need.

Reply:
this might be a good question for @bketelsen as he worked on the makefile for the uppercase demo. I'm just copying code here :)